configs/eslint/node/package.json

```diff
@@ -16,13 +16,13 @@
     "lint:js:fix": "prettier --write \"**/*.js\""
   },
   "dependencies": {
-    "@babel/core": "^7.22.9",
-    "@babel/eslint-parser": "7.22.9",
-    "@rushstack/eslint-patch": "^1.3.2",
-    "@typescript-eslint/eslint-plugin": "^6.2.0",
-    "@typescript-eslint/parser": "^6.2.0",
-    "eslint-config-prettier": "^8.9.0",
-    "eslint-import-resolver-typescript": "^3.5.5",
+    "@babel/core": "^7.22.10",
+    "@babel/eslint-parser": "7.22.10",
+    "@rushstack/eslint-patch": "^1.3.3",
+    "@typescript-eslint/eslint-plugin": "^6.3.0",
+    "@typescript-eslint/parser": "^6.3.0",
+    "eslint-config-prettier": "^9.0.0",
```

Comment:
The main change seemed to be supporting the flat configuration that will be available in `eslint@v9`.

https://github.com/prettier/eslint-config-prettier/blob/v9.0.0/CHANGELOG.md#version-900-2023-08-05